### PR TITLE
Feature/813 Add configuration tests with checkov

### PR DESCRIPTION
**Pull request by trottomv**

### CODE REVIEW

**Review by niccolomineo — APPROVED** [review]

**Comment by pauloxnet** [comment]
See comments in the backend PR

File: {{cookiecutter.project_dirname}}/.checkov.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+evaluate-variables: true
+output: [cli]
+output-file-path: .checkov
+quiet: true
+skip-framework: [gitlab_ci,kubernetes,sca_image,sca_package,terraform_plan]
+soft-fail: true
+skip-path: [
+    pacts,
+    tests,
+]
```
Comment (trottomv) [inline {{cookiecutter.project_dirname}}/.checkov.yaml:12]: add EOF

File: requirements/test.in [hunk 2]
```diff
@@ -1,5 +1,7 @@
 -r common.in
+black~=22.3.0
 coverage[toml]~=6.4.0
+checkov~=2.2.0
```
Comment (filippo-20tab) [inline requirements/test.in:4]: For the moment, let's remove this dependancy and allow developers to launch checkov checks via its Make command. It must be launched externally, but it is an acceptable compromise, not to have a whole load of requirements installed in the local container.

**Review by niccolomineo — APPROVED** [review]

**Comment by trottomv** [comment]
consider it a spike for now, postponed

**Pull request closed without merging**